conf/test.config

```diff
@@ -20,7 +20,8 @@ params {
   // To be build with: `nextflow run build.nf --build -profile docker --outdir references`
   igenomesIgnore = true
   genome = 'smallGRCh37'
-  genomes_base = 'references'
+  genomes_base = "https://github.com/MaxUlysse/nf-core_test-datasets/raw/sarek/reference"
```

Comment:
This will need to be changed to `https://github.com/nf-core/test-datasets/raw/sarek/reference`, once the related PR is done and merged.

Reply:
```suggestion
  genomes_base = "https://github.com/nf-core/test-datasets/raw/sarek"
```